src/components/movieTile/__tests__/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import MovieTile from '..';
+import userEvent from '@testing-library/user-event';
+
+const movieName = 'Bohemian Rhapsody';
+const releaseYear = '2003';
+const genres = ['Drama', 'Biography', 'Music'];
+const handleClick = jest.fn();
+const handleEdit = jest.fn();
+const handleDelete = jest.fn();
+
+const setup = () => {
+  const user = userEvent.setup();
+  const utils = render(
+    <MovieTile
+      movieName={movieName}
+      releaseYear={releaseYear}
+      imageUrl='someurl'
+      genres={genres}
+      onClick={handleClick}
+      onEdit={handleEdit}
+      onDelete={handleDelete}
+    />
+  );
+  return {
+    user,
+    ...utils,
+  };
+};
+
+describe('Movie Tile', () => {
+  it('Should render movie name', () => {
+    setup();
+
+    expect(screen.getByText(movieName)).toBeTruthy();
```

Comment:
To check that it's present in the doc it's better to use assert `toBeInTheDocument`

Reply:
replaced in all places